resources/src/main/java/org/robolectric/res/android/LoadedArsc.java

```diff
@@ -697,9 +696,11 @@ static LoadedPackage Load(Chunk chunk,
             if (builder_ptr != null) {
               builder_ptr.AddType(type);
             } else {
-              logError(String.format(
-                  "RES_TABLE_TYPE_TYPE with ID %02x found without preceding RES_TABLE_TYPE_SPEC_TYPE.",
-                  type.id));
+                logError(
```

Comment:
This format looks like not correct.

Reply:
@hellosagar This code formatting looks like incorrect too.

Reply:
my bad, I'm trusting the formatter blindly.  I'll format it manually.